config/settings.yml

```diff
@@ -318,3 +318,6 @@ discourse_ai:
   ai_bot_add_to_header:
     default: true
     client: true
+  ai_bot_github_access_token:
```

Comment:
This is a bit trickier with GitHub I guess because there are _so many_ things you can do on their API with a key. Not really necessary to do anything here, just a thought.